File: _archive/archive/Lua/lca/maths/random.lua
random = function (min, max)
    max, min =
    max or min or 1,
    max and min or 0

    return math.random() * (max-min) + min
end

randomInt = function (min, max)
    max, min =
    max or min or 1,
    max and min or 0
    
    return math.floor(
        random(
            math.floor(min),
            math.floor(max)+1))
end

seed = math.randomseed
seed(tonumber(tostring(os.time()):reverse():sub(1,6)))

File: _archive/cc/engine/random.js
let random = Math.random;

function randomInt(min, max) {
    if (max)
        return Math.floor(random() * (max - min) + min);
    else
        return Math.floor(random() * min);
}

function randomColor() {
    return color(random(), random(), random());
}

function randomPoint() {
    return new createVector(randomInt(width), randomInt(height));
}

function random2D(len = 1) {
    return p5.Vector.random2D().mult(len);
}

function noiseColor() {
    return color(random(), random(), random());
}

function noiseScale(x, y) {
    return noise(x * 0.005, y * 0.005);
}